content/en/dashboards/ddsql_editor/reference/scalar_functions.md

```diff
@@ -65,6 +79,9 @@ These functions return one value per row.
 | Name | Return type | Description | 
 |------|-------------|-------------| 
 | json_extract_path_text(text json, text path…) | text | Extracts the JSON sub-object in JSON as text, defined by the path. Its behavior is equivalent to the [postgres function with the same name][2]. For example, `json_extract_path_text(col, ‘forest')` returns the value of the key `forest` for each JSON object in `col`. See the example below for a JSON array syntax.|
+| json_extract_path(text json, text path…) | json | Same functionality as `json_extract_path_text`, but returns a column of type JSON instead of type text.|
```

Comment:
```suggestion
| json_extract_path(text json, text path…) | json | Same functionality as `json_extract_path_text`, but returns a column of JSON type instead of text type.|
```